### PR TITLE
allow reading ingress v1

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/skipper/rbac.yaml [hunk 1]
```diff
@@ -9,6 +9,13 @@ kind: ClusterRole
 metadata:
   name: skipper-ingress
 rules:
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs:
+  - get
+  - list
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/rbac.yaml:18]: IMO would be nice to use either yaml or json array notation consistently

**Comment by AlexanderYastrebov** [comment]
:+1:

File: cluster/manifests/skipper/rbac.yaml (hunk 1 above)
Comment (szuecs) [reply cluster/manifests/skipper/rbac.yaml:18]: yes I also thought the same and then I thought the part below we will drop after v1 migration...

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**